lib/query-builder.js

```diff
@@ -9,425 +9,442 @@ var operatorMap = {
   'and$': 'AND'
 }
 
-function parseExpression (entp, sTypes, current_name, current_value, params, values) {
-  var result = {
-    processed: true
-  }
+module.exports = function (opts) {
+  // var fromColumn = opts.fromColumnName ? opts.fromColumnName : fromColumnName
+  var toColumn = opts.toColumnName ? opts.toColumnName : toColumnName
```

Comment:
`_.identity` instead of `toColumnName`? I think it should work and you get read of the dummy function definition